core/src/main/scala/higherkindness/mu/rpc/srcgen/compendium/CompendiumMode.scala

```diff
@@ -1,46 +1,52 @@
 package higherkindness.mu.rpc.srcgen.compendium
 
 import java.io.{File, PrintWriter}
-import cats.effect.{Async, Resource, Sync}
+
+import cats.effect.{Async, ConcurrentEffect, Resource, Sync}
```

Comment:
```suggestion
import cats.effect.{Resource, ConcurrentEffect}
```
I think you have a mix of _effects_ here, but I believe that you are just requesting `ConcurrentEffect` for the client resource usage.